src/Rules/WebAPI/OrganizationalAuth/Single/Startup.cs

```diff
@@ -52,7 +52,7 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
             app.UseJwtBearerAuthentication(new JwtBearerOptions
             {
                 Authority = Configuration["Authentication:AzureAd:AADInstance"] + Configuration["Authentication:AzureAd:TenantId"],
-                Audience = Configuration["Authentication:AzureAd:AudienceUrl"]
```

Comment:
Yes, this should cover both read and non read, as currently for web api they are identical